### ISSUE TITLE
Channel was Force Closed within 24 hours.

### ISSUE THREAD

**Issue by cAP5L0CK** [MutinyWallet/mutiny-web#354]
I opened a channel yesterday by sending 250k sats to the wallet over lightning.  Channel opened immediately (very slick).  I had backed up my seed and am using a password.  Went to sleep.  When I woke up the wallet had Force Closed and the sats were swept into an onchain address.  I sent those back to myself.  (Not the best deal on a "loop out" but I will pretend that was what I did lol)

**Node(s):** 02fc3484fd01120ea13cf54e11bce10b2950d7d6f5fe70f171922e728b72e73505
**Peer(s):** {
  "pubkey": "03aefa43fbb4009b21a4129d05953974b7dbabbbfb511921410080860fca8ee1f0",
  "connection_string": "03aefa43fbb4009b21a4129d05953974b7dbabbbfb511921410080860fca8ee1f0@52.88.33.119:9735",
  "alias": null,
  "color": null,
  "label": null,
  "is_connected": true
}

**Currently no channels.**

**Transaction details:**
![image](https://github.com/MutinyWallet/mutiny-web/assets/11202204/492ee338-78b7-4450-88dd-bd9253a6bf40)

**Original Lightning receive transaction:**
{
  "__wbg_ptr": 4090552
}
![image](https://github.com/MutinyWallet/mutiny-web/assets/11202204/4ddd7433-a0fd-4c9b-af2e-6d57eae62fae)

**Channel Close:**
txid: 7ff19d1d7732af642bdc59861833a3198d75ee87bf31d3ca91f2682c5d08538e
![image](https://github.com/MutinyWallet/mutiny-web/assets/11202204/cc8493ca-e2eb-4361-986e-c8a0c959759b)

**Manual transaction back to my node:**
fa849cdf23cdaa223282f3420bb62a73a8ff237d3efee3da136fe8077a476dfc

What else do you need?

**Comment by benthecarman** [comment]
If you could include your logs that would help, it is located in settings > emergency kit. 

If you're on android it's possible it loaded pages in the background even if you didn't explicitly open the tab.

**Comment by cAP5L0CK** [comment]
[mutiny-logs.txt](https://github.com/MutinyWallet/mutiny-web/files/12052837/mutiny-logs.txt)

**Comment by cAP5L0CK** [comment]
My devices ARE an android phone (Browser DDG, and sometimes inbedded Chrome which is likely the way I first saw the app) and a (Arch) linux PC (firefox).  I did intentionally try to avoid using the phone since I was aware that  trying to get to the wallet on more than one device could be problematic.  But since the phone was the first place I opened the app, then it was hit from both devices.  My intention was to load the wallet with a bunch of sats and spew them all over Nostr over the weekend lol.
